Timeout now configurable

mhjmaas opened the pull request:
Made it possible to configure the default timeout by adding an
“df-selenium-tools.properties” file next to the jar file. This file
must contain the “default.wait.timeout = <msecs>” property.

Please note this property is only read when loading the jar file to
optimise performance. When there is no file present it will revert to
the default of 30 seconds

mhjmaas committed:
Timeout now configurable

Made it possible to configure the default timeout by adding an
“df-selenium-tools.properties” file next to the jar file. This file
must contain the “default.wait.timeout = <msecs>” property.

Please note this property is only read when loading the jar file to
optimise performance. When there is no file present it will revert to
the default of 30 seconds

wvanderdeijl commented:
Wouldn't a system property be much simpler? That's simpler to read and doesn't require all the file operations. Even if we would go for a properties file, I would prefer a simple load from classpath.
But my first option would be a simple system property.

mhjmaas commented:
Hi Wilfred,

I'll have a look at the system property approach. Hold off on the pull
request for a moment then.

Gr. Marcel

2015-11-09 15:24 GMT+01:00 Wilfred van der Deijl <notifications@github.com>:

> Wouldn't a system property be much simpler? That's simpler to read and
> doesn't require all the file operations. Even if we would go for a
> properties file, I would prefer a simple load from classpath.
> But my first option would be a simple system property.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/wvanderdeijl/adf-selenium/pull/20#issuecomment-155077541>
> .
>

mhjmaas committed:
Modified the timeout to a system property

The default.wait.timeout can now be set as a system property and if set
will use this value. Otherwise it will revert to the original. Also
added the AdfPanelCollection component

mhjmaas commented:
Hi Wilfred,

I've updated the code to use a system property. Also I have added another component.

Greetings,

Marcel

mhjmaas committed:
Merge remote-tracking branch 'wvanderdeijl/master' into timeout
